packages/terra-table/src/TableCell.jsx

```diff
@@ -12,11 +12,14 @@ const propTypes = {
 const TableCell = ({
   content,
   ...customProps
-}) => (
-  <td {...customProps} data-class="cell">
-    {content}
-  </td>
-);
+}) => {
+  const dataClassNames = customProps['data-class'] ? `cell ${customProps['data-class']}` : 'cell';
```

Comment:
Is there a discussion thread on the decision to introduce these data attributes?